desktop_qt_ui/widgets/editor_toolbar.py

```
@@ -138,17 +145,238 @@ def _init_ui(self):
+    # ------------------------------------------------------------------
+    _ICON_SZ = 28
+
+    @staticmethod
+    def _make_align_icon(mode: str, rect_color: QColor, line_color: QColor) -> QIcon:
+        """绘制对齐图标（28×28）。视觉重心居中。"""
+        s = 28
+        px = QPixmap(s, s)
+        px.fill(Qt.GlobalColor.transparent)
+        p = QPainter(px)
+        p.setRenderHint(QPainter.RenderHint.Antialiasing, False)
+        sr = 8; lr = 13; rt = 5; gap = 6     # 短/长方块宽、厚度、间距
```

Ruff (0.15.13)

[error] Multiple statements on one line (semicolon)

(E702)